core/payment/src/cli.rs

```diff
@@ -17,27 +17,39 @@ use crate::{wallet, DEFAULT_PAYMENT_DRIVER};
 #[derive(StructOpt, Debug)]
 pub enum PaymentCli {
     Accounts,
-    Enter {
-        amount: String,
+    // TODO: Uncomment when operation is supported by drivers
+    // Enter {
+    //     #[structopt(long)]
+    //     account: Option<String>,
+    //     #[structopt(long, default_value = DEFAULT_PAYMENT_DRIVER)]
+    //     driver: String,
+    //     #[structopt(long)]
+    //     network: Option<String>,
+    //     #[structopt(long)]
+    //     amount: String,
+    // },
+    Exit {
```

Comment:
pls add `///` comments to fill `--help` for all commands

Reply:
Good idea. Will do if there's enough time.